test/helper.js

```diff
@@ -1,10 +1,12 @@
 var assert = require('assert');
-var debug = require('debug')('strong-pm:test');
 var path = require('path');
 var util = require('util');
 
 require('shelljs/global');
 
+// XXX should be a way to ignore specific shelljs globals
```

Comment:
I added a fixup commit that removes this... it actually found a real leaked global on line 122!